backend/application/syson-application/pom.xml

```diff
@@ -81,48 +81,13 @@
 		</dependency>
 		<dependency>
 			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-starter</artifactId>
+			<artifactId>sirius-web-starter</artifactId>
 			<version>${sirius.web.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-web-graphql</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-web-services</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-web-spring</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-domain-edit</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-domain-design</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-view-diagram-edit</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-view-builder</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius.emfjson</groupId>
-			<artifactId>org.eclipse.sirius.emfjson</artifactId>
-			<version>2.3.7-SNAPSHOT</version>
+			<groupId>org.eclipse.syson</groupId>
+			<artifactId>syson-application-configuration</artifactId>
```

Comment:
Why do you add `syson-application-configuration` here? It is already existing below.

Reply:
Done